http/server.lua

```diff
@@ -742,7 +754,43 @@ local function parse_request(req)
     return p
 end
 
+local function add_connection(self, fiber_obj)
+    local fiber_id = fiber_obj:id()
+    self.connections[fiber_id] = {
+        obj = fiber_obj,
+        last_time_active = fiber.clock(),
+    }
+
+    return true
+end
+
+local function close_connection(self, fiber_obj, fiber_id, logger)
+    -- Name of fiber for http connection contains a hostname and port of http client.
+    local fiber_name = fiber_obj:name()
+    self.connections[fiber_id] = nil
+    logger("Close connection with %s, idle_timeout was exceeded\n", fiber_name)
```

Comment:
Not sure that fiber_name will say anything. Probably we could put in message `"%s (%d) - %s", fiber.name(), fiber.id(), peer`

Reply:
Fiber name that processes connection in `socket.tcp_server` ([documentation](https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-connect)) is not a plain string, but a host IP address and port. See snippet below:

```lua
sergeyb@pony:~/sources/MRG/http$ cat fiber_name.lua 
local socket = require('socket')
local fiber = require('fiber')

local function handler()
    print('Fiber name is', fiber.self():name())
end
socket.tcp_server('localhost', 8000, handler)
socket.tcp_connect('localhost', 8000)
sergeyb@pony:~/sources/MRG/http$ tarantool fiber_name.lua 
started
Fiber name is   server/127.0.0.1:44634
entering the event loop
```